docs/resources/dsm_app.md

```diff
@@ -6,13 +6,58 @@ Returns the Fortanix DSM App from the cluster as a resource
 
 ## Usage Reference
 
+
+
 ```
+
+variable "app_data" {
+    type = any
+    default = {
+        "other_group" = {
```

Comment:
what is an "other_group"?

Reply:
other_group_permissions = "${var.app_data.other_group}"

It defines the new group and it's permissions.
It is an user-defined, user can change it based on his comfort.
It is named as other group as **other_group_permissions** is indirectly relates to **other_group** in the original terraform body.